build.gradle

```diff
@@ -147,6 +147,11 @@ subprojects {
     tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
         // TODO: Re-enable when we fixed all those warnings.
         //kotlinOptions.allWarningsAsErrors = true
+        kotlinOptions {
+            freeCompilerArgs += [
+                "-Xuse-experimental=kotlin.ExperimentalUnsignedTypes",
```

Comment:
I don't suppose we can use the version of Kotlin where these are no longer experimental?

Reply:
Ah good point, This is needed for the UIntArray stuff used somewhere. I'm not sure why the in-code annotations didn't work. I'll add another comment so we remember to double-check this.